blackbox/docs/best_practice/rolling_upgrade.txt

```diff
@@ -268,5 +269,5 @@ Step 4: Repeat
 
 Repeat step two, three, and four for all other nodes.
 
-.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
 .. _accompanying notes: https://crate.io/a/upgrade-your-cluster-to-0-55/
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
```

Comment:
this is reference in the text. check out the top of the "Upgrade Process" section